react/features/face-landmarks/FaceLandmarksHelper.ts

```diff
@@ -150,14 +156,32 @@ export class HumanHelper implements FaceLandmarksHelper {
 
     getFaceExpression({ detections }: Detection): string | undefined {
         if (detections[0]?.emotion) {
-            return  FACE_EXPRESSIONS_NAMING_MAPPING[detections[0]?.emotion[0].emotion];
+            return FACE_EXPRESSIONS_NAMING_MAPPING[detections[0]?.emotion[0].emotion];
+        }
+    }
+
+    getAge({ detections }: Detection): number | undefined {
+        return detections[0]?.age;
```

Comment:
why don't we use the results of all detections?

Reply:
I wanted to get the age only for one person, because the same is happening with expression